src/Mod/Web/Gui/BrowserView.cpp

```diff
@@ -237,65 +222,25 @@ Py::Object BrowserViewPy::setHtml(const Py::Tuple& args)
  */
 
 WebView::WebView(QWidget *parent)
-    : QWEBVIEW(parent)
+    : QWebEngineView(parent)
 {
-    // Increase html font size for high DPI displays

```

Comment:
Why is this whole block removed? The function setTextSizeMultiplier() is used for the QtWebEngine.